gtest/src/lib.rs

```diff
@@ -508,8 +508,24 @@ pub mod constants {
 
     /// Maximal amount of reservations program may have.
     pub const MAX_RESERVATIONS: u64 = 256;
-    /// Cost of wasm module instantiation before execution per byte of code.
-    pub const MODULE_INSTANTIATION_BYTE_COST: Gas = 20;
+    /// Cost of wasm module code section instantiation before execution per byte
```

Comment:
these should be similar to real weights. please dump them as we do for instructions (or open an issue for asap handling)